.github/workflows/dev-test-run.yml

```
@@ -0,0 +1,66 @@
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: "20"
```

actionlint (1.7.4)

expecting a single ${{...}} expression or array value for matrix variations, but found plain text node

(syntax-check)